src/gui/folderwizard.cpp

```diff
@@ -141,6 +144,31 @@ void FolderWizardLocalPath::slotChooseLocalFolder()
     emit completeChanged();
 }
 
+
+void FolderWizardLocalPath::changeEvent(QEvent *e)
+{
+    switch (e->type()) {
+    case QEvent::StyleChange:
+    case QEvent::PaletteChange:
+    case QEvent::ThemeChange:
+        // Notify the other widgets (Dark-/Light-Mode switching)
+        emit styleChanged();
+        break;
+    default:
+        break;
+    }
+
+    QWizardPage::changeEvent(e);
+}
+
+void FolderWizardLocalPath::slotStyleChanged()
+{
+    const QColor warnYellow = Theme::isDarkColor(QGuiApplication::palette().base().color()) ? QColor(63, 63, 0) : QColor(255, 255, 192);
```

Comment:
do not repeat yourself
use `const auto warnYellow`

Reply:
Sure, changed